connect_Raspi/comp.py

```diff
@@ -5,84 +5,52 @@
 
 GPIO.setmode(GPIO.BCM)
 
-# pin used in mode 1
-GPIO.setup(20, GPIO.IN, pull_up_down=GPIO.PUD_DOWN)
-GPIO.setup(23, GPIO.OUT)
+IN=[20,21,24]
+LED=[23,18,17]
+BPM=[80,116,145]
```

Comment:
[IMO]
下のコード(16, 17行目等)で書かれているように配列の`,`の後には空白一文字入れて表記を合わせたほうが読みやすいかもです〜
あとは `=` の周りに空白入れても良さそうです！

```suggestion
IN = [20, 21, 24]
LED = [23, 18, 17]
BPM = [80, 116, 145]
```

https://pep8-ja.readthedocs.io/ja/latest/#id16